roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable=missing-docstring
+
+"""Interface to OpenShift oc command"""
+
+import os
+import shlex
+import shutil
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+ADDITIONAL_PATH_LOOKUPS = ['/usr/local/bin', os.path.expanduser('~/bin')]
+
+
+def locate_oc_binary():
+    ''' Find and return oc binary file '''
```

Comment:
https://www.python.org/dev/peps/pep-0008/#string-quotes

Indeed PEP8 doesn't make a recommendation, but one common approach is to follow what the Python REPL does: single quotes most of the time, double quotes when the string contains a single quote.

Now that said, it's likely overkill to go and change everything in one shot. And because we have no automated way to apply a standard / enforce it, there's little hope to achieve perfect consistency in this area ;)

My original comment was more in the intention to share a Python practice.